src/backbone-metal.js

```diff
@@ -88,10 +88,18 @@ function _wrapAll(dest, source) {
  * @memberOf Backbone
  */
 var Class = Metal.Class = function() {
+  this.cid = _.uniqueId(this.cidPrefix);
   this.initialize(...arguments);
 };
 
 /**
+ * Identifier prefix of the cid. It can be override.
+ * @public
+ * @attributeOf Class {String}
+ */
+Class.prototype.cidPrefix = 'bmc';
```

Comment:
`metal` is probably a better one here, even I had to think about what `bmc` meant